src/jloptions.c

```diff
@@ -106,6 +107,8 @@ static const char opts[]  =
     " --handle-signals={yes*|no} Enable or disable Julia's default signal handlers\n"
     " --sysimage-native-code={yes*|no}\n"
     "                            Use native code from system image if available\n"
+    " --pkgimage-native-code={yes*|no}\n"
+    "                            Use native code from package images if available ($)\n"
```

Comment:
I am not sure if this description is appropriate here (please excuse if it is): As far as I understand, `--pkgimage-native-code` does not just control if native code is to be used "if available" (since, in contrast to the sysimage, package images are not static over the lifetime of a Julia session), but also whether new code will be cached upon precompilation, right? That is, shouldn't this flag read something like
```suggestion
    "                            Enable or disable native code caching during precompilation and/or package loading ($)\n"
```
If this were to be changed here, this should also be changed in the documentation for the command line flags.